db/migrate/20240402230156_restore_donation_sites.rb

```diff
@@ -0,0 +1,11 @@
+class RestoreDonationSites < ActiveRecord::Migration[7.0]
+  def up
+    DonationSite.all.each do |site|
+      site.update!(active: true)
+    end
```

Comment:
This works no problem. FYI in the future - it's a lot faster to use `update_all`.